packages/components/organisms/f-checkout/src/components/_tests/helpers/setup.js

```diff
@@ -117,12 +118,22 @@ const $logger = {
     logError: jest.fn()
 };
 
+// eslint-disable-next-line
+const mockAuthToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.'
```

Comment:
Super ultra nit: this is not really a "mock", given we're not mocking any behaviour. Maybe we should rename this? Don't worry too much about it, though.